.vscode/settings.json

```diff
@@ -6,7 +6,19 @@
     "cmake.configureSettings": {
         "POWER_GRID_MODEL_SANITIZER": 1
     },
-    "testMate.cpp.test.executables": "${command:cmake.buildDirectory}/bin/power_grid_model_*_tests",
+    "testMate.cpp.test.executables": "${command:cmake.buildDirectory}/bin/power_grid_model_*_tests.exe",
```

Comment:
`.exe` does not exist on unix. please make sure both windows and unix are supported